hafnian/__init__.py

```diff
@@ -83,10 +79,16 @@
     hafnian_repeated,
     reduction,
 )
+from ._hermite_multidimensional import hafnian_batched, hermite_multidimensional
 from ._permanent import perm, perm_complex, perm_real, permanent_repeated
 from ._torontonian import tor
 from ._version import __version__
 
+if platform.system() == "Windows":  # pragma: no cover
+    extra_dll_dir = os.path.join(os.path.dirname(__file__), ".libs")
+    if os.path.isdir(extra_dll_dir):
+        os.environ["PATH"] += os.pathsep + extra_dll_dir
+
```

Comment:
`isort` probably moved this down, but this will have to be moved back to before the line
```python
from ._hafnian import (
```

to make sure it imports correctly on windows :)

Reply:
@bgupt can you look into this?